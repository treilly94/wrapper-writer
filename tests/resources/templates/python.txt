{%- set name = method.get("name") -%}
{%- set params = method.get("params") -%}
def {{ name }}({% for p in params.keys() %}{{ p }}{% endfor %}):
    api = jvm.TestFunc

    DataFrame(api.{{ name }}({% for p in params.keys() %}{{ p }}{% endfor %}), sql_ctx)