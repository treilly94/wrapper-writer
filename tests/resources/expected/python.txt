def testFunc(param1):
    """ A cool function"""

    api = jvm.TestFunc

    DataFrame(api.testFunc(param1), sql_ctx)